### PR TITLE
Add several spelling corrections

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -6382,6 +6384,7 @@ completly->completely
 completness->completeness
 complette->complete
 complettly->completely
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:6386]: completd->complete, completed,
incompletd->incomplete
* https://github.com/search?q=completd&type=code
* https://github.com/search?q=incompletd&type=code

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -12394,7 +12397,9 @@ exectuables->executables
 exectued->executed
 exectuion->execution
 exectuions->executions
+execture->execute
 exectured->executed
+exectures->executes
 exectute->execute
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:12403]: execturing->executing
* https://github.com/search?q=execturing&type=code

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -30512,6 +30527,7 @@ ue->use, due,
 ueful->useful
 uegister->unregister
 uesd->used
+uesful->useful
 uggly->ugly
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:30531]: uesfull->useful
uesfulness->usefulness
uesless->useless
ueslessness->uselessness
unsless->unless, useless,
  * https://github.com/search?q=uesfull&type=code
  * https://github.com/search?q=uesfulness&type=code
  * https://github.com/search?q=uesless&type=code
  * https://github.com/search?q=ueslessness&type=code
  * https://github.com/search?q=unsless&type=code

**Review by vikivivi — CHANGES_REQUESTED** [review]
That will be great if you could add these as well.

**Comment by luzpaz** [comment]
Added in a0957a1

**Review by vikivivi — APPROVED** [review]
LGTM

**Pull request merged**